### ISSUE TITLE
Failure in Importing CSV Exported from Linear

### ISSUE THREAD

**Issue by yoiki** [linear/linear#479]
Hello,
I've encountered an issue while importing a CSV file that was exported from Linear. 

The CSV file follows the format below:
```
"ID","Team","Title","Description","Status","Estimate","Priority","Project ID","Project","Creator","Assignee","Labels","Cycle Number","Cycle Name","Cycle Start","Cycle End","Created","Updated","Started","Triaged","Completed","Canceled","Archived","Due Date","Parent issue","Roadmaps","Project Milestone ID","Project Milestone","SLA Status"
```

However, it seems that the import process is failing due to the CSV parsing in [this line](https://github.com/linear/linear/blob/1305f73216aba778fdd313755e3ebe35cc7789fa/packages/import/src/importers/linearCsv/LinearCsvImporter.ts#L50) trying to parse the CSV using a semicolon (`;`) instead of a comma (`,`).

This discrepancy appears to be causing the error. I believe adjusting the parsing to match the actual CSV format should resolve this issue.

Thank you for your attention to this matter. I'm looking forward to your feedback or any potential solutions.
Kind regards,

**Comment by guillaumelachaud** [comment]
Hi @yoiki,

This has been fixed in [@linear/import@1.10.2](https://github.com/linear/linear/releases/tag/%40linear%2Fimport%401.10.2). You can update to this version now. Please let me know if you still have any issues.

**Comment by yoiki** [comment]
@guillaumelachaud 

Thank you for the quick response and for addressing the issue. I will update @linear/import@1.10.2 and check if the issue has been resolved. I appreciate your assistance and will reach out if I encounter any other issues.
Best regards,
